tests/installation/installation_overview.pm

```diff
@@ -62,6 +62,29 @@ sub ensure_ssh_unblocked {
     }
 }
 
+sub check_default_target {
+    # Check the systemd target where scenario make it possible
+    return if (is_caasp || is_upgrade || is_hyperv ||
+        get_var('REMOTE_CONTROLLER') || (get_var('BACKEND', '') =~ /spvm|pvm_hmc/));
+    # exclude non-desktop environment as well
+    return if (!get_var('DESKTOP') || get_var('PATTERNS'));
```

Comment:
As an alternative, which will be more safe, we could introduce new variable to trigger hard fail.
Could you please add bug reference to the comments, so it's clear why we skip case when we edit package selection.

Reply:
The problem with that alternative is that we will need to check again those boxes in whole OSD/O3 and took me a lot of time (we should discuss for next time we do this, other alternative, likes logging all the hits in some common place).
I will go for the comment then.

Reply:
fixed